grype-server/pkg/scanner/scanner.go

```diff
@@ -133,12 +135,27 @@ func (s *Scanner) ScanSbomJson(sbom string) (*models.Document, error) {
 }
 
 func (s *Scanner) scan(packagesContext grype_pkg.Context, packages []grype_pkg.Package) (*models.Document, error) {
-	store := store.Store{
+	vulnerabilityMatcher := createVulnerabilityMatcher(store.Store{
 		Provider:          s.vulProvider,
 		MetadataProvider:  s.vulMetadataProvider,
 		ExclusionProvider: s.exclusionProvider,
+	})
+
+	allMatches, ignoredMatches, err := vulnerabilityMatcher.FindMatches(packages, packagesContext)
+	// We can ignore ErrAboveSeverityThreshold since we are not setting the FailSeverity on the matcher.
+	if err != nil && errors.Is(err, grypeerr.ErrAboveSeverityThreshold) {
```

Comment:
shouldn't be !errors.Is(err, grypeerr.ErrAboveSeverityThreshold)?

Reply:
oops, tnx.